akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -73,8 +80,12 @@ private[akka] class Association(
     if (queue eq null)
       queue = Source.queue(256, OverflowStrategy.dropBuffer)
         .to(transport.outbound(this)).run()(materializer)
-    if (systemMessageQueue eq null)
-      systemMessageQueue = Source.queue(256, OverflowStrategy.dropBuffer)
-        .to(transport.outboundSystemMessage(this)).run()(materializer)
+    if (systemMessageQueue eq null) {
+      val (q, control) = Source.queue(256, OverflowStrategy.dropBuffer)
+        .toMat(transport.outboundControl(this))(Keep.both)
+        .run()(materializer)
+      systemMessageQueue = q
+      _outboundControlIngress = control
```

Comment:
unfortunately not, because those are `vars` (unless there is another syntax that I don't know about)